ckanext/sse/auth.py

```
@@ -26,37 +28,49 @@ def data_reuse_update(context, data_dict):
-    if submission_id:
+    user_obj = context.get('auth_user_obj')
+    if submission_id and user_obj:
         try:
-            submission = FormResponse.get(submission_id)
-            if submission and submission.user_id == context.get('auth_user_obj').id:
+            submission = FormResponse.get(submission_id, include_all=True)
+            if submission and submission.user_id == user_obj.id:
                 return {'success': True}
-        except:
+        except Exception:
             pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)